modules/document_repository/php/module.class.inc

```diff
@@ -26,4 +26,38 @@ namespace LORIS\document_repository;
  */
 class Module extends \Module
 {
+    /**
+     * {@inheritDoc}
+     *
+     * @param \User $user The user whose access is being checked.
+     *
+     * @return bool whether access is granted
+     */
+    public function hasAccess(\User $user) : bool
+    {
+        if (parent::hasAccess($user) == false) {
+            return false;
+        }
+        return $user->hasPermission('document_repository_view');
```

Comment:
```suggestion
        return $user->hasAnyPermission(
            ['document_repository_view', 'document_repository_delete']
        );
```

Reply:
I modeled the hasAccess functions for modules that were missing it on the `LorisMenuPermissions` values in `SQL\0000-00-02-Menus.sql`.. this isn't the current behaviour.

Reply:
It matches the hasAccess function for the module though. I think the intended behaviour is that someone with delete permissions should be able to view -- how else would they delete?

If it wasn't in the SQL table and showing up in the menu then I think that was an oversight. No harm fixing it here, imo.

Reply:
This *is* the `hasAccess` function for the module.. but I just checked and you're right, it doesn't match any of the other pages, so I guess we can fix it here.